### PR TITLE
*: `flashback to timestamp` delete experimental flag

### PR DESCRIPTION
**Pull request by Defined2014**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: sql-statements/sql-statement-flashback-to-timestamp.md [hunk 1]
```diff
@@ -9,8 +9,7 @@ TiDB v6.4.0 引入了 `FLASHBACK CLUSTER TO TIMESTAMP` 语法，其功能是将
 
 > **警告：**
 >
-> - 当前该功能为实验特性，不建议在生产环境中使用。
-> - 在执行 `FLASHBACK CLUSTER TO TIMESTAMP` 之前，需要暂停 PITR 和 TiCDC 等工具上运行的同步任务，待 `FLASHBACK` 执行完成后再启动，否则会造成同步失败等问题。
+> - `FLASHBACK CLUSTER TO TIMESTAMP` 的原理是用最新的时间戳重写一份旧的数据，并不会直接删除数据，所以在使用前请确保集群有足够的存储空间。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-flashback-to-timestamp.md:12]:
```suggestion
> **注意：**
>
> `FLASHBACK CLUSTER TO TIMESTAMP` 的原理是用最新的时间戳重写一份旧的数据，并不会直接删除数据，所以在使用前请确保集群有足够的存储空间。
```

“足够的存储空间”描述比较模糊，可以说明推荐/至少的存储空间大小吗？例如至少需要为集群 x2？

Comment (Defined2014) [reply sql-statements/sql-statement-flashback-to-timestamp.md:12]: 很难给一个具体的数值，最坏的情况是要有现在写入数据一半的空间，但直接写一半感觉很吓人，具体的写入的数据量要根据 flashback 这段时间的写入量来估算。

File: sql-statements/sql-statement-flashback-to-timestamp.md [hunk 2]
```diff
@@ -7,10 +7,9 @@ summary: TiDB 数据库中 FLASHBACK CLUSTER TO TIMESTAMP 的使用概况。
 
 TiDB v6.4.0 引入了 `FLASHBACK CLUSTER TO TIMESTAMP` 语法，其功能是将集群的数据恢复到特定的时间点。
 
-> **警告：**
+> **说明：**
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-flashback-to-timestamp.md:10]:
```suggestion
> **注意：**
```

File: sql-statements/sql-statement-flashback-to-timestamp.md [hunk 3]
```diff
@@ -7,10 +7,9 @@ summary: TiDB 数据库中 FLASHBACK CLUSTER TO TIMESTAMP 的使用概况。
 
 TiDB v6.4.0 引入了 `FLASHBACK CLUSTER TO TIMESTAMP` 语法，其功能是将集群的数据恢复到特定的时间点。
 
-> **警告：**
+> **注意：**
 >
-> - 当前该功能为实验特性，不建议在生产环境中使用。
-> - 在执行 `FLASHBACK CLUSTER TO TIMESTAMP` 之前，需要暂停 PITR 和 TiCDC 等工具上运行的同步任务，待 `FLASHBACK` 执行完成后再启动，否则会造成同步失败等问题。
+> `FLASHBACK CLUSTER TO TIMESTAMP` 的原理是用最新的时间戳重写一份旧的数据，并不会直接删除数据，所以在使用前请确保集群有足够的存储空间。
```
Comment (TomShawn) [inline sql-statements/sql-statement-flashback-to-timestamp.md:12]:
```suggestion
> `FLASHBACK CLUSTER TO TIMESTAMP` 是用最新的时间戳重写入一份旧的数据，并不会直接删除当前数据，所以在使用前请确保集群有足够的存储空间。
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: sql-statements/sql-statement-flashback-to-timestamp.md [hunk 4]
```diff
@@ -7,10 +7,9 @@ summary: TiDB 数据库中 FLASHBACK CLUSTER TO TIMESTAMP 的使用概况。
 
 TiDB v6.4.0 引入了 `FLASHBACK CLUSTER TO TIMESTAMP` 语法，其功能是将集群的数据恢复到特定的时间点。
 
-> **警告：**
+> **注意：**
 >
-> - 当前该功能为实验特性，不建议在生产环境中使用。
-> - 在执行 `FLASHBACK CLUSTER TO TIMESTAMP` 之前，需要暂停 PITR 和 TiCDC 等工具上运行的同步任务，待 `FLASHBACK` 执行完成后再启动，否则会造成同步失败等问题。
+> `FLASHBACK CLUSTER TO TIMESTAMP` 是用最新的时间戳重写入一份旧的数据，并不会直接删除当前数据，所以在使用前请确保集群有足够的存储空间。
```
Comment (TomShawn) [inline sql-statements/sql-statement-flashback-to-timestamp.md:12]:
```suggestion
> `FLASHBACK CLUSTER TO TIMESTAMP` 是用最新的时间戳写入特定时间点的旧数据，但不会删除当前数据，所以在使用前请确保集群有足够的存储空间来同时容纳旧数据和当前数据。
```

**Pull request merged**